### ISSUE TITLE
Backend Track: Observation on the curiculum

### ISSUE THREAD

**Issue by udofia2** [3MTT-PET/Proficiency-Tool#3]
Considering there is frontend module for the same study duration for each student in web development, i think having CSS flex box might not be necessary in the  backend curriculum.

Secondly, while curating questions for the entry level, should HTML and CSS be included in the backend questions?

![flexBoxCss](https://github.com/3MTT-PET/Proficiency-Tool/assets/45747099/4ab7249d-1fc2-45f4-8c13-e3f182e6547c)

**Comment by udofia2** [comment]
still under backend, there is introduction to Nextjs, I'm guessing the intention was Nestjs
![nextjs](https://github.com/3MTT-PET/Proficiency-Tool/assets/45747099/ee8c6a45-d128-4caa-9824-6a01cf539d3c)

**Comment by udofia2** [comment]
Also considering that Nestjs is included in the curriculum, should the curated questions include Nestjs?.

**Comment by 3MTT** [comment]
> Considering there is frontend module for the same study duration for each student in web development, i think having CSS flex box might not be necessary in the backend curriculum.
> 
> Secondly, while curating questions for the entry level, should HTML and CSS be included in the backend questions?
> 
> ![flexBoxCss](https://private-user-images.githubusercontent.com/45747099/315272487-4ab7249d-1fc2-45f4-8c13-e3f182e6547c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTExMDQ1ODEsIm5iZiI6MTcxMTEwNDI4MSwicGF0aCI6Ii80NTc0NzA5OS8zMTUyNzI0ODctNGFiNzI0OWQtMWZjMi00NWY0LThjMTMtZTNmMTgyZTY1NDdjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIyVDEwNDQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwMWJiNTAyOGVmYTU2MzFhOTFlMmZlZTg4NjljYWI2OGQ2YTZhNTg2YTNmODAzZmRmMzA3ZTZjYzQ2MDkyZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hoBBVR6K1WP96bw1cUFvA_ltM19KeGkY-exuUYQGqNw)

I agree that CSS flexbox in a backend course might be an overreach. You may leave out HTML and CSS for backend questions since it is expected to be covered in the frontend.

**Comment by 3MTT** [comment]
> still under backend, there is introduction to Nextjs, I'm guessing the intention was Nestjs ![nextjs](https://private-user-images.githubusercontent.com/45747099/315276375-ee8c6a45-d128-4caa-9824-6a01cf539d3c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTExMDQ1NzMsIm5iZiI6MTcxMTEwNDI3MywicGF0aCI6Ii80NTc0NzA5OS8zMTUyNzYzNzUtZWU4YzZhNDUtZDEyOC00Y2FhLTk4MjQtNmEwMWNmNTM5ZDNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIyVDEwNDQzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM4OGRlMThhZDIwNzAzOGVlY2ZhODYzYzRhMjhiMGU4YjMyMzg0YzQ1MzkxMzg4NTBkY2Q2OGEwOTkzNTE0NDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jdytHNcj1nw4Xaokpx9_nchdWq8Es5I0nAM096t6jGA)

Right! It should be Nest.js. But I think that it could be Next.js too since it is full-stack framework.
